Week5/solution-week-5.py

```diff
@@ -0,0 +1,32 @@
+import csv
+
+with open('student.csv','w',newline='')as f:
+    w=csv.writer(f)
+    header=['sid','sname','city','contact']
+    w.writerow(header)
+
+#Insert 5 Records Directly
+    row=[[1,'om','kedarnath',9876567890],
+         [2,'sai','shirdi',1231231230],
+         [3,'ram','ayodhya',1234567865],
+         [4,'kishan','vrundavan',2345670763],
+         [5,'ravan','lanka',8978655544]]
+    w.writerows(row)
+
+
+#Insert 5 Records Take Input From User 

```

Comment:
If i want to take 5 records from user using writerows( ) instead of writerow( ) as you use in your code,
so what should i do?

Reply:
import csv

# Initialize an empty list to store user records
user_records = []

# Take input for 5 records from the user
for i in range(5):
    name = input("Enter Name: ")
    age = int(input("Enter Age: "))
    location = input("Enter Location: ")
    user_records.append([name, age, location])

# Open a CSV file in write mode
with open('user_records.csv', 'w', newline='') as csvfile:
    csvwriter = csv.writer(csvfile)
    
    # Use writerows() to write all user records
    csvwriter.writerows(user_records)

print("User records have been written to CSV file.")